darts/tests/utils/tabularization/test_create_lagged_training_data.py

```diff
@@ -17,6 +17,25 @@
 from darts.utils.timeseries_generation import linear_timeseries
 
 
+def helper_create_multivariate_linear_timeseries(
+    n_components: int, components_names: Sequence[str] = None, **kwargs
+) -> TimeSeries:
+    """
+    Helper function that creates a `linear_timeseries` with a specified number of
+    components. To help distinguish each component from one another, `i` is added on
+    to each value of the `i`th component. Any additional keyword arguments are passed
+    to `linear_timeseries` (`start_value`, `end_value`, `start`, `end`, `length`, etc).
+    """
+    timeseries = []
+    if components_names is None or len(components_names) < n_components:
```

Comment:
what if `len(components_names) > n_components`?

it seems that just the first `n_components` are considered. Is this the behaviour expected? Or an error should be raised?

Reply:
This is an helper function for the tests, if `components_names > n_components`, the last values will just be ignored.